### ISSUE TITLE
bug: facing SQL initialization error when connected to neon DB (PostgresSQL)

### ISSUE THREAD

**Issue by meetzaveri** [authorizerdev/authorizer#295]
**Version:** 
`1.1.28-rc.1` (src is pointing to - https://github.com/authorizerdev/authorizer-render)

**Describe the bug**

I have my authorizer service on render. Have my postgres DB hosted by neon.tech. Since few days, my service is failing with error
```
Failed to initialize SQL driver: simple protocol queries must be run with standard_conforming_strings=on
```
**Steps To Reproduce**

1. Create postgres DB in neon
2. Create authorizer service on render
3. Connect authorizer service to that DB by supplying necessary credentials
4. Deploy service or do manual deploy

**Expected behavior**

It shouldn't throw SQL initialization error as previously it was working as expected, I think a week or 2 before. I saw these errors this week only.

**Screenshots**
<img width="1436" alt="render_authorizer_logs_18_11_2022" src="https://user-images.githubusercontent.com/21293048/202678923-ed85293c-89af-461d-9fa1-bed3f2df4c4d.png">

**Desktop (please complete the following information):**

- OS: macOS
- Browser chrome
- Version

**Comment by meetzaveri** [comment]
related to #276

**Comment by lakhansamani** [comment]
Blocked because of neno-db issue (https://github.com/neondatabase/neon/issues/2752)

**Comment by lakhansamani** [comment]
@meetzaveri this is resolved in [#1.1.38](https://github.com/authorizerdev/authorizer/releases/tag/1.1.38)
